src/components/map/MapboxMap.tsx

```diff
@@ -42,25 +43,32 @@ function mapOnLoad(map: mapboxgl.Map, dispatch: Dispatch<any>) {
   });
 }
 
-const MapboxGLMap = (): any => {
+interface MapboxMapProps {
+  mapConfig?: any
+  countriesConfig: CountriesMapConfig,
+  studyPinsConfig: StudyPinsMapConfig
+}
+
+const MapboxMap = ( {mapConfig, countriesConfig, studyPinsConfig}: MapboxMapProps ): any => {
   const mapContainerRef = useRef<HTMLDivElement>(null);
   const [state, dispatch] = useContext(AppContext);
   const [map, setMap] = useState<mapboxgl.Map | undefined>(undefined);
 
   // Creates map, only runs once
   useEffect(() => {
     (async () => {
-      const baseMapStyle = await getEsriVectorSourceStyle(MapUrlResource.WHO_BASEMAP);
+      const baseMapStyle = await getEsriVectorSourceStyle(MapResources.WHO_BASEMAP);
 
-      const m = new mapboxgl.Map({
-        //@ts-ignore
-        container: mapContainerRef.current,
-        style: baseMapStyle,
-        center: [10, 30],
-        zoom: 2,
-        attributionControl: false,
-        antialias: true// enables MSAA antialiasing, to help make dotted line WHO borders more visible
-      });
+      const options = { // Merges options together to configure map
```

Comment:
more descriptive variable name like `mergedMapOptions` could replace this comment